### ISSUE TITLE
Problems with dropbox, I cannot route mouse to actual object

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#5257]
**Reported by levilibegas on 2015-08-01 03:17**
Hi there, I am using nvda very last snap version, firefox the last version and windows xp, though it has happened the same  with windows 7. I access to my page in Dropbox using firefox. There is a four elements list in which I can upload a file, create a folder, share a folder and let show deleted files. These options are in a four elements list. I cannot let nvda access these elements or options. Sometimes, but I cannot know how and why I happened by accident to access upload a new file... When I tried to move by object, and inside the list with these four elemnts, I tried to route mouse pointer to any of these options,but I couldn't because the mouse would only get to the search field which is just before those elements. Can you see the same scenario, do you have the same problemas, do you have any workaround or approach? Thanks for your help. Levilibegas.

**Comment by Adriani90** [comment]
The interface of Dropbox webpage has changed significantly. I can access those buttons without issue. @levilibegas:, if you are still having this issue please feel free to comment on it. I will close it as works for me but We can reopen it if the initial author reports it again. Thanks.
